var/spack/repos/builtin/packages/py-gxformat2/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyGxformat2(PythonPackage):
+    """Galaxy Workflow Format 2 Descriptions"""
+
+    homepage = "https://github.com/galaxyproject/gxformat2"
+    pypi = "gxformat2/gxformat2-0.16.0.tar.gz"
+
+    version("0.16.0", sha256="16ff5aae1456e0a65c1ed644537e66626ea047a567c8eda19d2a2eb0b20fb752")
+
+    depends_on("python@3.6:", type=("build", "run"))
+    depends_on("py-setuptools", type="build")
+
```

Comment:
Can you add a comment saying that the tarball is missing the `requirements.txt` file but you can find it on GitHub?

Reply:
done!